### PR TITLE
Handle html content correctly in access restrictions 

### LINKED ISSUE #954: Access restrictions chips show escaped html
**What happened**:
The Access restrictions chips show escaped html as text. 
image

**What you expected to happen**:
Do not escape html comping from the backend. 

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know**:

**Environment**:

### PR DESCRIPTION
**Pull request by holgerkoser**
**What this PR does / why we need it**:
With this PR html content in access restrictions input and display components is handled correctly.

**before (description and title as text)**:

**after (description as v-html)**:

**padding between title and subtitle**:

**Which issue(s) this PR fixes**:
Fixes #954 

**Special notes for your reviewer**:
I have removed markdown processing for titles in the backend. For all others frontend configuration texts we do also not process titles but only descriptions. I think it is more consistent now. 

**Release note**:

```bugfix user
HTML content is correctly displayed in access restriction components
```

### CODE REVIEW

File: frontend/src/components/ShootAccessRestrictions/AccessRestrictions.vue [hunk 1]
```diff
@@ -31,8 +34,14 @@ SPDX-License-Identifier: Apache-2.0
               ></v-checkbox>
             </v-list-item-action>
             <v-list-item-content>
-              <v-list-item-title class="wrap-text" :class="textClass(definition)" v-html="transformHtml(optionValue.input.title)"></v-list-item-title>
-              <v-list-item-subtitle v-if="optionValue.input.description" class="wrap-text pt-n3" :class="textClass(definition)" v-html="transformHtml(optionValue.input.description)"></v-list-item-subtitle>
+              <v-list-item-title class="wrap-text" :class="textClass(definition)">
```
Comment (holgerkoser) [inline frontend/src/components/ShootAccessRestrictions/AccessRestrictions.vue:37]: If there is a description add space between title and description

Comment (holgerkoser) [reply frontend/src/components/ShootAccessRestrictions/AccessRestrictions.vue:37]: ✔️

**Review by petersutter — APPROVED** [review]
/lgtm

**Review by grolu — APPROVED** [review]
/lgtm

**Pull request merged**